generators/add-extension/index.js

```diff
@@ -108,7 +108,7 @@ class Generator extends Base {
     const file = this.destinationPath(pathToRegistry);
 
     const old = this.fs.read(file);
-    const absFile = d.module.startsWith('~') ? d.module.slice(1) : `./src/${d.module.includes('.') ? d.module.slice(0, d.module.lastIndexOf('.')) : d.module}`;
+    const absFile = d.module.startsWith('~') ? d.module.slice(1) : `./${d.module.includes('.') ? d.module.slice(0, d.module.lastIndexOf('.')) : d.module}`;
```

Comment:
The solution works only for TS files, but not for JS files. At the moment we are still supporting the phovea.js files (see line 113). Please change the code that the file path still works for both cases. If it is easier and better to read, you can refactor it like so:

```ts
if(fs.existsSync('src/phovea.ts')) {
  const absFile = '...';
  const importFunction =  ' ...';
} else {
  const absFile = '...';
  const importFunction =  ' ...';
}
```